sol-transactions/templates/proto/mydata.proto.gotmpl

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package mydata.v1;
+
+import 'sf/solana/type/v1/type.proto';
+
+message MyData {
+  repeated .sf.solana.type.v1.ConfirmedTransaction transactions = 1;
```

Comment:
Can we have something smaller, this will output a lot of data, can we find something that emits less data somehow?